css/ai.css

```diff
@@ -32,8 +32,25 @@
         @apply fixed drawer-button lg:hidden z-0 bg-primary text-primary-content top-2 left-2 btn-square w-fit h-fit p-1 rounded-md !important
     }
 
-    #header_img {
-        @apply max-h-full max-w-full sm:max-h-80 sm:mx-auto
+    #header-bg {
+        width: 100%;
+        /* aspect-ratio: 3 / 1; */
+        height: 305px;
+        display: flex;
+        background: url('/img/ai/ai-header-bg.jpg') no-repeat center right;
+        background-size: cover;
+        align-items: center;
+    }
+
+    #header-content {
+        /* @apply max-h-full max-w-full sm:max-h-80 sm:mx-auto */
```

Comment:
If these styles aren't being used any long do they need to be kept? For this line and other comments throughout this file